src/Writers/HTMLWriter.jl

```diff
@@ -650,6 +650,9 @@ function render(doc::Documents.Document, settings::HTML=HTML())
         for filename in readdir(joinpath(ASSETS, "js"))
             path = joinpath(ASSETS, "js", filename)
             endswith(filename, ".js") && isfile(path) || continue
+            if filename == "warner.js" && !doc.user.warn_outdated
+                continue
+            end
             push!(r, JSDependencies.parse_snippet(path))
```

Comment:
Is the `.js` handled properly when it's loaded via RequireJS?

Reply:
It is, yes. I'd kinda like to load it a bit earlier, but it shouldn't matter in practice.